pkg/tfbridge/provider_test.go

```diff
@@ -5177,7 +5177,7 @@ func TestPlanResourceChangeUnknowns(t *testing.T) {
 				"listProps":null,
 				"nestedListProps":null,
 				"maxItemsOneProp":null,
-				"setBlockProps":[{"prop":""}],
+				"setBlockProps":"04da6b54-80e4-46f7-96ec-b56ff0331ba9",
```

Comment:
I haven't yet done the digging to figure out why only sets are affected by this bug - lists seem fine.

Reply:
Maybe intentional I think, if a set has any unknown elements, they might merge with other elements once known, it's tricky, so might as well mark the entire thing as unknown. Could be intentional behavior.

Reply:
Well I take that back, the previous result had empty string which is suspect..